dev/change_version.py

```diff
@@ -248,6 +249,24 @@ def change_project_number_Doxyfile(file, new_version):
     f.write(changed_str)
     f.close()
 
+def change_project_number_readme(file, new_version):
+    changed_str = ""
+
+    markers = ["<Reference", "<HintPath>"]
+    #                             0   .  15etc
+    version_regex = re.compile(r'[0-9]\.[0-9]+')
+
+    f = open(file, 'r')
+    for line in f:
+        if any(marker in line for marker in markers):
+            changed_str += version_regex.sub(new_version, line, count=1)
+        else:
+            changed_str += line
+    f.close()
+
+    f = open(file, 'w')
+    f.write(changed_str)
+    f.close()
```

Comment:
You can do everything in one shot, e.g.
```python
lines = re.sub(r"((?:<Reference |<HintPath>).*?)\d\.\d+",
               "\\1" + new_version, open(file).read(), flags=re.IGNORECASE | re.MULTILINE)

open(file, 'w').write(lines)
```
(assuming that the version has `9.999` format). Also, for `open()/close()` calls, you can use `with open(file, 'w') as f: f.write()` although it is not necessary in this particular case (the flie will be closed as soon as `f` goes out of scope anyway)